.storybook/utils.tsx

```diff
@@ -15,28 +15,28 @@ import {
   text
 } from "@storybook/addon-knobs";
 
-import colors from "../node_modules/@esri/calcite-colors/colors.json";
+import { Colors } from "../node_modules/@esri/calcite-colors/dist/colors";
```

Comment:
Unrelated: Should this be lowercased? I'd expect a class if uppercased.

Reply:
Agreed